### DISCUSSION TITLE
Proposal for modification of Name this list placeholder

### DISCUSSION THREAD

**Discussion by zkffhtm6523** [community/community#9292] [Other Feature Feedback, Questions, & Ideas]
![스크린샷 2021-12-25 오전 9 56 49](https://user-images.githubusercontent.com/66704969/147375246-0b3e896d-0098-4d2e-840f-09a13e12dedc.png)

I think Emoji and placeholder text are overlapped on m1 MacBook. So I'd like to suggest adding space more.

**Comment by brianlovin** [comment]
Interesting, what browser are you using here? I can't reproduce that:

<img width="483" alt="Screen Shot 2022-01-04 at 18 01 42@2x" src="https://user-images.githubusercontent.com/1923260/148148903-9ffb5c98-a0ce-409b-991a-97740cefab5f.png">
